code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/Identity.java

```diff
@@ -24,7 +24,9 @@
 
 import static com.adobe.marketing.mobile.edge.identity.IdentityConstants.LOG_TAG;
 
-
+/**
+ * Defines the public interface for the AEP Edge Identity extension.
```

Comment:
This may get confused with the java interface meaning. Maybe reword to..
`Defines the public APIs..`